src/main/java/com/comphenix/protocol/wrappers/PlayerInfoData.java

```diff
@@ -139,7 +174,16 @@ public Object getGeneric(PlayerInfoData specific) {
 					Object gameMode = EnumWrappers.getGameModeConverter().getGeneric(specific.gameMode);
 					Object displayName = specific.displayName != null ? specific.displayName.handle : null;
 
-					if (MinecraftVersion.WILD_UPDATE.atOrAbove()) {
+					if (MinecraftVersion.FEATURE_PREVIEW_UPDATE.atOrAbove()) {
+						return constructor.newInstance(
+								specific.profileId,
+								specific.profile.handle,
+								specific.listed,
+								specific.latency,
+								gameMode,
+								displayName,
+								 null); // TODO: do we want to support this?
```

Comment:
typically i'll do it if it's easy, otherwise i'll wait until someone requests it